.github/workflows/integrity-checks.yml

```diff
@@ -0,0 +1,39 @@
+name: Integrity Checks
+
+on:
+  push:
+    branches:
+      - main
+      - release-*
+  pull_request:
+    branches:
+      - main
+      - release-*
+
+jobs:
+ verify_generated_files:
+    name: Perform API Integrity Check
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+          fetch-depth: 0
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.x
+      - name: Install prerequisites and download drivers
+        shell: pwsh
+        run: .\build.ps1 -prereqs -driver
+      - name: Read commit SHA
+        id: read-upstream-commit-sha
+        run: echo "::set-output name=hash::$(cat src/Playwright/.drivers/linux/package/commitinfo)"
+      - name: Checkout submodule
+        run: git checkout ${{ steps.read-upstream-commit-sha.outputs.hash }}
+        working-directory: ./playwright
+      - name: Generate API
+        run: .\build.ps1 -api
+        shell: pwsh
```

Comment:
what is pwsh?

Reply:
https://docs.github.com/en/actions/guides/building-and-testing-powershell